### PR TITLE
doc: Governance changes for Collaboration Network

### PR DESCRIPTION
**Pull request by mhdawson**
Governance changes to be landed once the Collaboration
network proposal reaches stage 3.

Signed-off-by: Michael Dawson <michael_dawson@ca.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,12 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
+support of the OpenJS foundation through a collaboration space, plear read our [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md) document, or contact any CPC member with questions.
```
Comment (kborchers) [inline README.md:68]: > plear read our

please read our

File: README.md [hunk 2]
```diff
@@ -142,9 +152,10 @@ Discussion should be held in the open whenever possible.  However, if you need t
 
 ### Project Resources & Travel Fund
 
-OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md).
+OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.
+
+OpenJS Foundation Collaboration spaces are able to take advantage of serveral services and benefits as outlined in [Collaboration Space Progression](./proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md).
```
Comment (kborchers) [inline README.md:157]: > serveral services and benefits

several services and benefits

File: README.md (hunk 1 above)
Comment (mhdawson) [inline README.md:68]:
```suggestion
support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
```

File: README.md (hunk 2 above)
Comment (mhdawson) [inline README.md:157]:
```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration 
```

File: README.md [hunk 3]
```diff
@@ -61,6 +61,12 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
+support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
+
```
Comment (MylesBorins) [inline README.md:69]: Should there be a note here that there are none? It seems weird for it to mention `The following are official OpenJS Collaboration spaces` but then have nothing blow

File: README.md [hunk 4]
```diff
@@ -88,6 +94,10 @@ According to the [CPC Charter](https://github.com/openjs-foundation/cross-projec
 - Christopher Hiller (@boneskull, IBM)
 - Tobie Langel (@tobie, UnlockOpen)
 
+#### Collaboration space representatives
+
+Members of the Collaboration spaces at the `Core` stage may nominate a candiate for one of the voting seats on the CPC which represents the Collaboration spaces. Currently there are no spaces at the `Core` stage and therefore no representative.
```
Comment (MylesBorins) [inline README.md:99]: What is the `Core` phase? Where is the documentation for this?

File: README.md [hunk 5]
```diff
@@ -185,4 +196,3 @@ If you run your project's social media and would like the Foundation to share or
 
 For any other topics which aren't covered above, please email
 [operations@openjsf.org](mailto:operations@openjsf.org).
-
```
Comment (MylesBorins) [inline README.md:188]: Is there no longer a newline at the end of the document?

File: CPC-CHARTER.md [hunk 6]
```diff
@@ -156,6 +156,10 @@ ensuring:
       substantial changes before final approval by CPC.
    1. Voting members are responsible for approving funds for budgets delegated
       to the project.
+   1. Voting members are responsible for approving new `collaboration spaces` within the scope
+      for OpenJS Foundation set by the Governing Board, as outlined in the OpenJS Collaboration Network process.
```
Comment (MylesBorins) [inline CPC-CHARTER.md:160]: `within the scope for` reads a bit odd. Is it necessary to be explicit there? Could it be simplified?

```suggestion
   1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS 
   Collaboration Network process. 
```

File: README.md (hunk 4 above)
Comment (mhdawson) [reply README.md:99]: These are documented here https://github.com/openjs-foundation/cross-project-council/blob/master/proposals/stage-0/COLLABORATION_NETWORK/COLLABORATION_SPACE_PROGRESSION.md#core-collaboration-spaces

File: README.md (hunk 3 above)
Comment (mhdawson) [reply README.md:69]: @MylesBorins good catch, added.

File: README.md (hunk 5 above)
Comment (mhdawson) [reply README.md:188]: Looks like there is still one.

File: README.md (hunk 3 above)
Comment (mhdawson) [reply README.md:69]: added

File: README.md [hunk 7]
```diff
@@ -142,9 +154,10 @@ Discussion should be held in the open whenever possible.  However, if you need t
 
 ### Project Resources & Travel Fund
 
-OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md).
+OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.
+
+OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [Collaboration
```
Comment (christian-bromann) [inline README.md:159]: This sentence doesn't seem to be complete. I am not entirely sure which section it refers to.

Comment (mhdawson) [inline README.md:159]:
```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```

Comment (mhdawson) [reply README.md]: @christian-bromann thanks for catching that. Fixed although the link won't work until https://github.com/openjs-foundation/cross-project-council/pull/595 lands

File: README.md [hunk 8]
```diff
@@ -61,6 +61,14 @@ The following projects are official OpenJS Foundation projects. If you are inter
 * [Fastify](https://fastify.io)
 * [nvm](http://nvm.sh/)
 
+
+### OpenJS Collaboration spaces
+
+The following are official OpenJS Collaboration spaces. If you are interested in fostering collaboration with the 
+support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression]
```
Comment (timmywil) [inline README.md:68]: It seems this link is blank.

File: README.md [hunk 9]
```diff
@@ -142,9 +154,10 @@ Discussion should be held in the open whenever possible.  However, if you need t
 
 ### Project Resources & Travel Fund
 
-OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md).
+OpenJS Foundation projects and their community members are able to take advantage of several services and benefits, including the [Travel Fund Program](./project-resources/MEMBER_TRAVEL_FUND.md). Details can be found in the [/project-resources](./project-resources) directory.
+
+OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [OLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```
Comment (darcyclarke) [inline README.md:159]: Looks like a small typo: the link to the `COLLABORATION_NETWORK.md` doc is missing a "C" at the beginning (currently: `...OLLABORATION_NETW...`)

File: CPC-CHARTER.md [hunk 10]
```diff
@@ -156,6 +156,10 @@ ensuring:
       substantial changes before final approval by CPC.
    1. Voting members are responsible for approving funds for budgets delegated
       to the project.
+   1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS 
+   Collaboration Network process.
```
Comment (darcyclarke) [inline CPC-CHARTER.md:160]: @mhdawson Should we link this "Collaboration Network Process"? Might be helpful for anyone reading to navigate...

File: README.md (hunk 8 above)
Comment (darcyclarke) [reply README.md:68]: I think it's supposed to be: `[Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md)`

**Review by MylesBorins — APPROVED** [review]
generally LGTM

**Comment by MylesBorins** [comment]
We could likely split out the README changes into another PR that doesn't require board approval... but keeping it in the single PR and only sending the charter changes to the board seems simple enough

File: README.md (hunk 8 above)
Comment (mhdawson) [inline README.md:68]:
```suggestion
support of the OpenJS foundation through a collaboration space, please read our [Collaboration Space Progression](./COLLABORATION_SPACE_PROGRESSION.md)
```

File: README.md (hunk 9 above)
Comment (mhdawson) [inline README.md:159]:
```suggestion
OpenJS Foundation Collaboration spaces are able to take advantage of several services and benefits as outlined in [COLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md).
```

File: CPC-CHARTER.md (hunk 10 above)
Comment (mhdawson) [inline CPC-CHARTER.md:160]:
```suggestion
1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS
[COLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md) process.
```

File: CPC-CHARTER.md [hunk 11]
```diff
@@ -156,6 +156,10 @@ ensuring:
       substantial changes before final approval by CPC.
    1. Voting members are responsible for approving funds for budgets delegated
       to the project.
+1. Voting members are responsible for approving new `collaboration spaces` as outlined in the OpenJS
+[COLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md) process.
```
Comment (mhdawson) [inline CPC-CHARTER.md:160]:
``` suggestion
  [COLLABORATION_NETWORK.md](COLLABORATION_NETWORK.md) process.
```

**Comment by mhdawson** [comment]
@darcyclarke thanks for the review, think I have addressed your comments.

**Review by tobie — APPROVED** [review]
LGTM

**Review by dylans — APPROVED** [review]

**Comment by mhdawson** [comment]
PR Landed, so closing

**Comment by mhdawson** [comment]
oops reopening, should have landed

**Pull request merged**